python/ql/src/Security/CWE-285/PamAuthorization.ql

```diff
@@ -33,4 +34,5 @@ where
     acctMgmtCall = libPam().getMember("pam_acct_mgmt").getACall() and
```

Comment:
I'm not allowed to add a comment in the appropriate place, but there's a use of `API::moduleImport("ctypes.util")` on line 20 above that will need fixing.

Reply:
indeed, that probably what the merge conflict is also about :+1: